src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -994,8 +994,6 @@ public abstract partial class StringComparer : System.Collections.Generic.ICompa
         public static System.StringComparer FromComparison(System.StringComparison comparisonType) { throw null; }
         public int GetHashCode(object obj) { throw null; }
         public abstract int GetHashCode(string obj);
-        bool System.Collections.IEqualityComparer.Equals(object x, object y) { throw null; }
```

Comment:
I'm not sure about these two. We don't have an explicit implementation, but we do have an implicit one. Was this intentional to hide this APIs?